fedgov-cv-rgaencoder/src/main/java/gov/usdot/cv/rgaencoder/DDate.java

```diff
@@ -17,44 +17,44 @@
 package gov.usdot.cv.rgaencoder;
 
 public class DDate {
-    private Integer year;
-    private Integer month;
-    private Integer day;
+    private int year;
```

Comment:
Why is Integer changed to int?

Reply:
When getting the methods in JNI wrapper, Integer was presenting errors due to it being a class. Omkar provided feedback saying it should be int.